src/main/java/jenkins/plugins/http_request/HttpRequestGlobalConfig.java

```diff
@@ -0,0 +1,82 @@
+package jenkins.plugins.http_request;
+
+import hudson.Extension;
+import hudson.XmlFile;
+
+import java.io.File;
+import java.util.List;
+import java.util.ArrayList;
+
+import jenkins.model.GlobalConfiguration;
+import jenkins.model.Jenkins;
+
+import jenkins.plugins.http_request.auth.Authenticator;
+import jenkins.plugins.http_request.auth.BasicDigestAuthentication;
+import jenkins.plugins.http_request.auth.FormAuthentication;
+
+import net.sf.json.JSONObject;
+
+import org.kohsuke.stapler.DataBoundSetter;
+import org.kohsuke.stapler.StaplerRequest;
+
+@Extension
+public class HttpRequestGlobalConfig extends GlobalConfiguration {
+
+    private List<BasicDigestAuthentication> basicDigestAuthentications = new ArrayList<BasicDigestAuthentication>();
+    private List<FormAuthentication> formAuthentications = new ArrayList<FormAuthentication>();
+
+    public HttpRequestGlobalConfig() {
+        load();
+    }
+
+    @Override
+    public boolean configure(StaplerRequest req, JSONObject json)
+    throws FormException
+    {
+        req.bindJSON(this, json);
+        save();
+        return true;
+    }
+
+    public static HttpRequestGlobalConfig get() {
+        return GlobalConfiguration.all().get(HttpRequestGlobalConfig.class);
+    }
+
+    public String getHttpRequestHelpPath() {
+        return "/descriptor/"+HttpRequest.class.getName()+"/help";
```

Comment:
Odd; would it not be easier to move the relevant `help-*.html` to the resource path for `HttpRequestGlobalConfig`, where they would be found by default?